src/pages/Graph/SummaryPanelNode.tsx

```diff
@@ -360,10 +453,17 @@ export default class SummaryPanelNode extends React.Component<SummaryPanelPropTy
           )}
           <RpsChart
             label="HTTP - Outbound Request Traffic"
-            dataRps={this.state.requestCountOut}
-            dataErrors={this.state.errorCountOut}
+            dataRps={this.state.httpRequestCountOut}
+            dataErrors={this.state.httpErrorCountOut}
             hide={isServiceNode}
           />
+          {this.isIstioOutgoingCornerCase(node) && (
+            <>
+              <div>
+                <Icon type="pf" name="info" /> Traffic to istio-system not included. Use edge for details.
+              </div>
+            </>
+          )}
```

Comment:
Yikes! I see that the edge also doesn't have sparklines. I guess the edge also collects data from source telemetry.
But this message is OK. The edge, at least, will show in the rates-table the specific details.
This is just a comment :)

Reply:
We can chase istio-system corner cases for a long time, instead let's stop and wait for any negative feedback. Good testing.